src/main/java/net/atos/entng/calendar/controllers/TrustedUrlController.java

```diff
@@ -0,0 +1,141 @@
+                            .put(Field.STATUS, Field.KO);
+                    renderError(request, object);
+                });
+    }
+
+    @Post("/trusted")
+    @ApiDoc("Add a trusted url")
+    @SecuredAction(value = "", type = ActionType.RESOURCE)
+    @ResourceFilter(SuperAdminFilter.class)
+    @Trace(Actions.CREATE_TRUSTED)
+    public void createTrustedUrl(HttpServerRequest request){
+        RequestUtils.bodyToJson(request, body -> {
+            trustedUrlService.create(body)
+                    .onSuccess(res -> {
+                        JsonObject object = new JsonObject()
+                                .put(Field.STATUS, Field.OK);
+                        renderJson(request, object);
+                    })
+                    .onFailure(err -> {
+                        JsonObject object = new JsonObject()
+                                .put(Field.REASON, err.getMessage())
+                                .put(Field.STATUS,Field.KO);
+                        renderError(request, object);
+                    });
+        });
+    }
+
+    @Delete("/trusted/:idUrl")
+    @ApiDoc("Delete a trusted url by id")
+    @SecuredAction(value = "", type = ActionType.RESOURCE)
+    @ResourceFilter(SuperAdminFilter.class)
+    @Trace(Actions.DELETE_TRUSTED)
+    public void deleteTrustedUrl(HttpServerRequest request){
+        final String id = request.params().get(Field.IDURL);
+        trustedUrlService.delete(id)
+                .onSuccess(res -> {
+                    JsonObject object = new JsonObject()
+                            .put(Field.STATUS, Field.OK);
+                    renderJson(request, object);
+                })
+                .onFailure(err -> {
+                    JsonObject object = new JsonObject()
+                            .put(Field.REASON, err.getMessage())
+                            .put(Field.STATUS, Field.KO);
+                    renderError(request, object);
+                });
+    }
+
+    @Put("/trusted/:idUrl")
+    @ApiDoc("Update a trusted url by id")
+    @SecuredAction(value = "",type = ActionType.RESOURCE)
+    @ResourceFilter(SuperAdminFilter.class)
+    @Trace(Actions.UPDATE_TRUSTED)
+    public void updateTrustedUrl(HttpServerRequest request){
+        final String id = request.params().get(Field.IDURL);
+        RequestUtils.bodyToJson(request, body -> {
+            trustedUrlService.update(id, body)
+                    .onSuccess(res -> {
+                        JsonObject object = new JsonObject()
+                                .put(Field.STATUS, Field.OK);
+                        renderJson(request, object);
+                    })
+                    .onFailure(err -> {
+                        JsonObject object = new JsonObject()
+                                .put(Field.REASON, err.getMessage())
+                                .put(Field.STATUS, Field.KO);
+                        renderError(request, object);
```

Comment:
Ici tu donne l'information de l'erreur en retour. Cela peut remonter des infos à des mauvaises personnes qui auront juste a appeler l'API. Tu peux notée l'erreur dans le log si tu veux.